recipe/meta.yaml

```diff
@@ -1,20 +1,21 @@
-{% set version = "3.17.6" %}
+{% set version = "3.19.2" %}
 
 package:
   name: simplejson
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/s/simplejson/simplejson-{{ version }}.tar.gz
-  sha256: cf98038d2abf63a1ada5730e91e84c642ba6c225b0198c3684151b1f80c5f8a6
+  sha256: 9eb442a2442ce417801c912df68e1f6ccfcd41577ae7274953ab3ad24ef7d82c
 
 build:
   number: 0
   #trigger 1
   script:
     - export REQUIRE_SPEEDUPS=1  # [not win]
     - set REQUIRE_SPEEDUPS=1  # [win]
-    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True # [py<38]
```

Comment:
```suggestion
```
Upstream doesn't mention dropping support of earlier python versions.

Reply:
yeah, note that these py skips should always be implemented for reasons other than "our CI doesn't build this python version"